etc/helm/pachyderm/values.yaml

```diff
@@ -180,8 +180,22 @@ ingress:
       create: false
       crt: ""
       key: ""
+
 loki:
   enabled: true
+
+# locki-stack contains values that will be passed to the loki-stack subchart
```

Comment:
Should we add a note on how to set the storageclass here?

Reply:
added 👍